setup.py

```diff
@@ -143,6 +143,8 @@
 
 DELTA_REQUIRED = ["deltalake"]
 
+DOCLING_REQUIRED = ["docling>=2.23.0"]
```

Comment:
Can we create an NLP_REQUIRED with TORCH, DOCLING, and Milvus in it? 

The goal would be to have something OOTB that combines these. HuggingFace and Gensim would probably be useful too but can be added later.